lib/main_common.pm

```diff
@@ -1447,6 +1447,7 @@ sub load_extra_tests_desktop {
         if (gnomestep_is_applicable()) {
             # Setup env for x11 regression tests
             loadtest "x11/x11_setup";
+            loadtest "x11/exiv2";
```

Comment:
add condition that test is for opensuse only

Reply:
This code is already under a "if (check_var('DISTRI', 'opensuse'))", should I really double check it on the load_test?

Reply:
ok, correct